### PR TITLE
Update multiple-links snippet

### PR DESCRIPTION
**Pull request by thedmail**

### CODE REVIEW

File: compose/snippets/src/main/java/com/example/compose/snippets/text/TextSnippets.kt [hunk 1]
```diff
@@ -550,17 +550,30 @@ private object TextEffectiveStateManagement2 {
...
+            withLink(
+                LinkAnnotation.Url(
+                    "https://developer.android.com/jetpack/compose",
+                    TextLinkStyles(style = SpanStyle(color = Color.Green))
+                )
+            ) {
+                append("Compose guidance")
+                pop()
+                append(".")
+                pop()
```
Comment (riggaroo) [inline compose/snippets/src/main/java/com/example/compose/snippets/text/TextSnippets.kt]: You dont need the `pop` operations here, just move the non-linked text out of the withLink() {} lambda. 

```
Text(
        buildAnnotatedString {
            append("Go to the ")
            withLink(
                LinkAnnotation.Url(
                    "https://developer.android.com/",
                    TextLinkStyles(style = SpanStyle(color = Color.Blue))
                )
            ) {
                append("Android Developers ")

            }

            append("website, and check out the")
            withLink(
                LinkAnnotation.Url(
                    "https://developer.android.com/jetpack/compose",
                    TextLinkStyles(style = SpanStyle(color = Color.Green))
                )
            ) {
                append("Compose guidance")
            }
            append(".")
        }
    )
```

**Review by riggaroo — CHANGES_REQUESTED** [review]
Please remove pop operations.

File: compose/snippets/src/main/java/com/example/compose/snippets/text/TextSnippets.kt (hunk 1 above)
Comment (thedmail) [reply compose/snippets/src/main/java/com/example/compose/snippets/text/TextSnippets.kt]: Oh, nice, thanks.

**Comment by riggaroo** [comment]
> > Apply spotless / apply-spotless (pull_request) Failing after 1m
> 
> Not sure what to do about this.

fixed it for you

**Review by riggaroo — APPROVED** [review]

**Pull request merged**